protx-client/src/components/ProTx/components/maps/MainMap.jsx

```diff
@@ -407,6 +407,8 @@ function MainMap({
             // Simple zoom to point clicked and having fixed zoom level for counties
             // See https://jira.tacc.utexas.edu/browse/COOKS-54
             map.setView(e.latlng, 9);
+          } else {
+            map.setView(e.latlng, 11);
```

Comment:
just picked a closer zoom level for census tracts and added this ticket: https://jira.tacc.utexas.edu/browse/COOKS-290